torch/_dynamo/variables/nn_module.py

```diff
@@ -604,6 +604,11 @@ class UnspecializedNNModuleVariable(UserDefinedObjectVariable):
     """
 
     def __init__(self, value, **kwargs):
+        if type(value) is torch.jit._script.RecursiveScriptModule:
```

Comment:
This works around a failure in `PYTORCH_TEST_WITH_DYNAMO=1 python test/test_jit.py -k test_versioned_div_scalar_inplace` where before this PR, dynamo actually bails out compiling the test code due to some other reason (missing .forward attribute on scriptmodule's type, iirc), and then the test passes.  With this PR, tracing __call__ caused dynamo to actually break down (LOW dynamo stack assert hit).

We should probably dig deeper into what was going on there.  But it was quite a mess of complexity - dynamo actually tries to compile tons of functions relating to the saving and loading of jit modules, etc.  Not sure it's worth properly debugging.